server/services/premisesService.test.ts

```diff
@@ -87,6 +94,9 @@ describe('PremisesService', () => {
         localAuthorities: [localAuthority1, localAuthority2, localAuthority3],
         characteristics: [premisesCharacteristic1, premisesCharacteristic2, genericCharacteristic],
         probationRegions: [probationRegion1, probationRegion2, probationRegion3],
+        // While PremisesService simply returns the contents of a JSON file, this will be the complete list of 400+
+        // PDUs, even in our tests, so check for a subset of known real PDUs
+        pdus: expect.arrayContaining([pdu1, pdu2, pdu3]),
```

Comment:
This threw me a bit. Would it not be better to import the PDU JSON file and expect that, rather than building using the factory and expecting a subset. Even better (and I don't know if you can do this in Jest), could we even use `jest.mock('../../path/to/data.json')` and mock the output of the JSON file?

Reply:
Oh! Turns out you can do jest.mock('../../path/to/data.json'), I kinda assumed that wouldn't work. Was a little fiddly because of this - https://www.bam.tech/article/fix-jest-mock-cannot-access-before-initialization-error - but yeah, this is nicer

Reply:
Yeah, that's a doozy that one - I come back to that blog post a lot! 😆